jsapp/js/components/common/audioPlayer.scss

```diff
@@ -0,0 +1,120 @@
+@import "scss/_colors";
+
+@mixin rangeTrack() {
+  -webkit-appearance: none;
+  border-radius: 10px;
+  background: $kobo-gray-92;
+  height: 8px;
+}
+
+// In order for the hack fix to work, we have to hard code the slider length
+$hack-fix-slider-length: 300px;
+$hack-fix-slider-length-int: 300;
+
+
+/* HACK FIX: Chrome browsers et al don't have a `progress` pseudo class like
+ * Firefox or Edge, so we make our own
+ *
+ * Inspired by https://stackoverflow.com/questions/38095650/style-input-range-to-look-like-a-progress-bar
+ */
+@function makelongshadow() {
+  $val: 5px 0 0 0 $kobo-blue;
+
+  @for $i from 6 through $hack-fix-slider-length-int {
+    $val: #{$val}, #{$i}px 0 0 0px #{$kobo-gray-92};
+  }
+
+  @return $val;
+}
+
+.audio-player {
+  display: flex;
+  justify-content: center;
+
+  .audio-player__controls {
+    margin-right: 12px;
+    i.k-icon {
+      font-size: 48px;
+      border-radius: 4px;
+      cursor: pointer;
+
+      &:hover {
+        background-color: $kobo-gray-92;
+      }
+    }
+
+    i.k-icon-caret-right, .k-icon-pause {
+      color: $kobo-blue;
+    }
+  }
+
+  .audio-player__progress {
+    padding-top: 6px;
+    .audio-player__time {
+      display: flex;
+      justify-content: space-between;
+      width: $hack-fix-slider-length;
```

Comment:
So this is probably causing the player to not fit on mobile screen:
<img width="478" alt="Screenshot 2021-07-21 at 17 57 44" src="https://user-images.githubusercontent.com/2521888/126529075-e459b885-8c3d-48fe-a356-86e877972cc1.png">

Reply:
Added a bit of hack-fixing that should be improved later